src/os_esp32.cc

```diff
@@ -659,37 +668,80 @@ class HeapSummaryCollector {
     counts_[type]++;
   }
 
+  void identify_toit_processes() {
```

Comment:
I'd try to be more consistent with the naming. Not sure the `toit_` prefix helps much here.